app/models/night_finder.rb

```diff
@@ -0,0 +1,30 @@
+class NightFinder
+  def find_nights(night_params)
+    @_night_params = night_params
+
+    Night.all_for_couch(couch_id)
+      .between_check_in_check_out(check_in, check_out)
```

Comment:
Do we need some kind of check to make sure that a night has/hasn't already been reserved? Like a similar enum to reservations since a reservation is no longer required to create a night